tests/step-005.sh

```diff
@@ -15,14 +14,14 @@ cd /tmp/shared/hooks-005.git &&
 if echo "$EXTRA_INSTALL_ARGS" | grep -q "use-core-hookspath"; then
     echo 'n
 y
-/tmp/shared/hooks-005.git
+file:///tmp/shared/hooks-005.git
 ' | sh /var/lib/githooks/install.sh || exit 1
 
 else
     echo 'n
 n
 y
-/tmp/shared/hooks-005.git
+file:///tmp/shared/hooks-005.git
```

Comment:
What's the default shared hook type? Is it local?
If that's the case, some of these don't have to change, right?
It would be good to have a good mix of both local/global URL/non-URL for coverage.

Reply:
if no protocol  its treated as local.
I though about, only tests `non-protocol` stuff (local paths)  in step-117, to better distinguish this feature from the current implementation. I would leave the original tests with `file://` thats also the most common usecase -> with protocol

Reply:
> https://coveralls.io/builds/33289402 seems to be fine for me, is there anything missing from it? Looks like `cli.sh` is not there, not sure if that's related to this change? Otherwise I wouldn't be worried where we get the data from as long as it's there.

Hm... thats utter strange, why is cli.sh not there? I though we dispatch to the release folder for the cli as well ?
Hmm have to check that